### ISSUE TITLE
Does not catch KMS Key with Principal: "*" in Key Policy

### ISSUE THREAD

**Issue by JoseRolles** [stelligent/cfn_nag#335]
Even though the following Key Policy will allow any AWS account to use this KMS Key to Decrypt anything, cfn_nag does not flag it as a warning or failing violation.

```json
{
  "Resources": {
    "MyKey": {
      "Type": "AWS::KMS::Key",
      "Properties": {
        "EnableKeyRotation": true,
        "KeyPolicy": {
          "Version": "2012-10-17",
          "Statement": [
            {
              "Effect": "Allow",
              "Principal": "*",
              "Action": "kms:*",
              "Resource": "*"
            }
          ]
        }
      }
    }
  }
}
```

**Comment by erickascic** [comment]
cfn_nag doesn't currently have a rule for anything KMS::Key related except for the rotation bit.  Will put this on the backlog.  Thanks!

**Comment by erickascic** [comment]
released in 0.4.65
